_includes/projects.html

```diff
@@ -1,34 +1,36 @@
-<div class="grid-item">
-  {% if project.redirect %}
-  <a href="{{ project.redirect }}">
-  {% else %}
-  <a href="{{ project.url | relative_url }}">
-  {% endif %}
-    <div class="card hoverable">
-      {% if project.img %}
-        {% include figure.html 
-          path=project.img 
-          alt="project thumbnail" %}
-      {% endif %}
-      <div class="card-body">
-        <h2 class="card-title text-lowercase">{{ project.title }}</h2>
-        <p class="card-text">{{ project.description }}</p>
-        <div class="row ml-1 mr-1 p-0">
-          {% if project.github %}
-          <div class="github-icon">
-            <div class="icon" data-toggle="tooltip" title="Code Repository">
-              <a href="{{ project.github }}"><i class="fab fa-github gh-icon"></i></a>
+
```

Comment:
sorry, what happened with indentation in this file? seems like a bunch of extra white spaces got added a the begging of each line. let's fix this before merge.